namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "route53_zone" {
```

Comment:
could you make sure both route53 zones are named differently please ?

Reply:
Done